main.tf

```diff
@@ -6,7 +6,7 @@ locals {
 resource "aws_instance" "ext_tableau_linux" {
   count                       = var.environment == "prod" ? "1" : "1" # 2 in Prod (Green & Blue), 1 in NotProd (Green only)
   key_name                    = var.key_name
-  ami                         = data.aws_ami.ext_tableau_linux.id
+  ami                         = var.environment == "prod" ? data.aws_ami.ext_tableau_linux.id : data.aws_ami.ext_tableau_linux_np.id
```

Comment:
Why introduce a separate variable for NotProd AMI?

Reply:
I am not yet confident the userdata for ext tab will play nicely with the new ~Tab versio - so i want to have a lee way so if we need to deploy a new server in prod i  do not need to revert my change